Modules/_sqlite/connection.c

```diff
@@ -59,19 +59,21 @@ static void _pysqlite_drop_unused_cursor_references(pysqlite_Connection* self);
 static PyObject *
 new_statement_cache(pysqlite_Connection *self, int maxsize)
 {
-    PyObject *args[] = { PyLong_FromLong(maxsize), };
-    if (args[0] == NULL) {
+    PyObject *args[] = { NULL, PyLong_FromLong(maxsize), };
+    if (args[1] == NULL) {
         return NULL;
     }
     PyObject *lru_cache = self->state->lru_cache;
-    PyObject *inner = PyObject_Vectorcall(lru_cache, args, 1, NULL);
-    Py_DECREF(args[0]);
+    PyObject *inner = PyObject_Vectorcall(
+        lru_cache, args + 1, 1 | PY_VECTORCALL_ARGUMENTS_OFFSET, NULL);
```

Comment:
Could you please break lines according to PEP 7? I'm striving to keep all `sqlite3` changes PEP 7 compliant :)
Ditto for the other calls.

Reply:
That is, either:

```suggestion
    PyObject *inner = PyObject_Vectorcall(lru_cache, args + 1,
                                          1 | PY_VECTORCALL_ARGUMENTS_OFFSET,
                                          NULL);
```

or:

```suggestion
    size_t nargs = 1 | PY_VECTORCALL_ARGUMENTS_OFFSET;
    PyObject *inner = PyObject_Vectorcall(lru_cache, args + 1, nargs, NULL);
```